### PR TITLE
fix: force a compatible texture format for the platform

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Use GraphicsFormatUtility to get a compatible texture for every platform.

## Test
 No "Texture creation failed" should appear, check WebGL, Windows and Mac builds. Additionally, the infinite floor should look correct.

### CODE REVIEW

File: unity-renderer/Assets/DCLPlugins/MapTexturePlugin/MapTexturePlugin.cs [hunk 1]
```diff
@@ -52,7 +47,7 @@ private static async UniTaskVoid DownloadTexture(string url, BaseVariable<Textur
                 }
                 else
                 {
-                    Texture2D tex = new Texture2D(0, 0, TextureFormat.ETC2_RGBA8, false, true);
+                    Texture2D tex = new Texture2D(0, 0, GraphicsFormatUtility.GetGraphicsFormat(TextureFormat.RGBA32, false), TextureCreationFlags.None);
```
Comment (mikhail-dcl) [inline unity-renderer/Assets/DCLPlugins/MapTexturePlugin/MapTexturePlugin.cs:50]: Could not we just use `textureFormat` instead? It was the only oversight here I believe?

Comment (AjimenezDCL) [reply unity-renderer/Assets/DCLPlugins/MapTexturePlugin/MapTexturePlugin.cs:50]: What I dont like about that approach is that we rely on a fixed set of textures formats and it doesnt scale that much. With this one we just ask the system to return a compatible GraphicsFormat with the needs we have.

Comment (mikhail-dcl) [reply unity-renderer/Assets/DCLPlugins/MapTexturePlugin/MapTexturePlugin.cs:50]: Can we delete `PRIORITIZED_FORMATS` then?

**Review by mikhail-dcl — APPROVED** [review]

**Review by kwssbocian — APPROVED** [review]
✔️No issues observed during testing. Additionally no occurrences of "Texture creation failed" error.

**Pull request merged**